doc/flame/examples/lib/remove_effect.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:doc_flame_examples/flower.dart';
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/experimental.dart';
+import 'package:flame/game.dart';
+import 'package:flutter/cupertino.dart';
+
+class RemoveEffectGame extends FlameGame with HasTappableComponents {
+  @override
+  Future<void> onLoad() async {
+    final flower = Flower(
+      position: size / 2,
+      size: 45,
+      onTap: (flower) {
+        flower.add(
+          RemoveEffect(delay: 3),
+        );
+      },
+    )..anchor = Anchor.center;
+
+    add(
+      SpawnBox(
+        onTap: () {
+          if (!children.contains(flower)) {
+            add(
+              flower
+                ..opacity = 0
+                ..add(
+                  OpacityEffect.by(
+                    1,
+                    EffectController(duration: 0.75),
+                  ),
+                ),
+            );
+          }
+        },
+      )
+        ..anchor = Anchor.topLeft
+        ..size = size,
+    );
```

Comment:
I would just use `TapDetector` on the game instead and use that to control the taps by overriding `onTapDown`, and I don't think the `OpacityEffect` is needed, better keep the code as short as possible for the example.

Reply:
Possibly having a `TextComponent` that shows how long it is left until it will be removed?